datasource/http/data.go

```diff
@@ -0,0 +1,153 @@
+//go:generate packer-sdc struct-markdown
+//go:generate packer-sdc mapstructure-to-hcl2 -type DatasourceOutput,Config
+package http
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"mime"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/hashicorp/hcl/v2/hcldec"
+	"github.com/hashicorp/packer-plugin-sdk/common"
+	"github.com/hashicorp/packer-plugin-sdk/hcl2helper"
+	packersdk "github.com/hashicorp/packer-plugin-sdk/packer"
+	"github.com/hashicorp/packer-plugin-sdk/template/config"
+	"github.com/zclconf/go-cty/cty"
+)
+
+type Config struct {
+	common.PackerConfig `mapstructure:",squash"`
+	// Url where should be getting things from
```

Comment:
Keeping it consistent with the terraform docs
```suggestion
	// The URL to request data from. This URL must respond with a `200 OK` response and a `text/*` or `application/json` Content-Type
```